manim/mobject/mobject.py

```diff
@@ -14,21 +18,11 @@
 from functools import partialmethod, reduce
 from math import ceil
 from pathlib import Path
-from typing import (
-    TYPE_CHECKING,
-    Callable,
-    Dict,
-    Iterable,
-    List,
-    Optional,
-    Sequence,
-    Tuple,
-    Type,
-    TypeVar,
-    Union,
-)
+from typing import TYPE_CHECKING, Any, Callable, Mapping, Sequence, TypeVar, Union, cast
 
 import numpy as np
+
+# TODO(types): colour has no type annotations.
```

Comment:
Would be fixed by either #1972 or #2050

Reply:
I don't think so. This is about [this colour](https://pypi.org/project/colour/) not the Manim `Color` type.

Reply:
Though you have just made me realise that I put `Color` in loads of places where I think it should have been `Colors`. I'll fix it.